love/src/Utils.js

```diff
@@ -100,22 +100,27 @@ export default class ManagerInterface {
     this.connectionIsOpen = false;
   }
 
+  static httpProtocol = window.location.protocol;
+  static wsProtocol = window.location.protocol === 'https:' ? 'wss:' : 'ws:';
```

Comment:
Could substitute `httpProtocol` here as you just defined it the way you are using it here.